orion/pkg/controllers/class_controller_test.go

```diff
@@ -0,0 +1,539 @@
+package controllers_test
+
+import (
+	"bytes"
+	"database/sql"
+	"encoding/json"
+	"errors"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/domains"
+	"github.com/ahsu1230/mathnavigatorSite/orion/pkg/services"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"net/http"
+	"testing"
+	"time"
+)
+
+//
+// Test Get All
+//
+func TestGetAllClasses_Success(t *testing.T) {
+	now := time.Now().UTC()
+	classService.mockGetAll = func() ([]domains.Class, error) {
+		return []domains.Class{
+			{
+				Id:         1,
+				ProgramId:  "program1",
+				SemesterId: "2020_spring",
+				ClassKey:   sql.NullString{String: "class1", Valid: true},
+				ClassId:    "program1_2020_spring_class1",
+				LocationId: "churchill",
+				Times:      "3 pm - 5 pm",
+				StartDate:  now,
+				EndDate:    now.Add(time.Hour * 24 * 60),
```

Comment:
You can just create two variables:
```
now := time.Now().UTC()
later := now.Add(...)
```
And just use these two variables instead of having to keep re-calculating the EndDate